components/builder-api/habitat/default.toml

```diff
@@ -5,13 +5,21 @@ listen = "0.0.0.0"
 port   = 9636
 
 [github]
+enabled        = false
```

Comment:
My thought with setting both of them to `enabled = false` by default was since we have multiple providers now, all should be disabled by default and you should specifically enable the one you need.  I imagine we have a `user.toml` or similar for overriding the default GitHub values with our production values, so couldn't we just put `enabled = true` in there?

Reply:
It's worth noting that we'll hit this ![bug](https://github.com/habitat-sh/habitat/issues/4576) on upgrade